javascript/general_style_guide.md

```diff
@@ -106,7 +117,7 @@ with its pros and cons. More info in the [React Docs](https://reactjs.org/docs/h
 
 ### Naming
 
-- Files: snake_case
+- Files: cameCase
```

Comment:
```suggestion
- Files: camelCase
```

Reply:
Wait, so this means that when we have a component `ClusterDashboardItem` the fiel name should be `clusterDashboardItem.js` instead of `ClusterDashboardItem.js`?

Reply:
I've have added more text, hope it makes sense now.